src/main/java/org/nuxeo/segment/io/SegmentIODataWrapper.java

```diff
@@ -111,8 +103,8 @@ protected boolean isAllowed(Object value) {
             if (key.startsWith(GROUP_KEY_PREFIX)) {
                 String gKey = key.substring(GROUP_KEY_PREFIX.length());
                 Serializable value = metadata.get(key);
-                if (value!=null) {
-                    if ( isAllowed(value)) {
+                if (value != null) {
+                    if (isAllowed(value)) {
```

Comment:
yeah, I could, but it's not related to my changes, I would'nt know how to name the method and the gain would be very poor (and I have to port it to 4 branches :D )